packages/sdk/src/react/StoryBoost/embed.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import gql from "graphql-tag";
+import { ApolloProvider, Mutation, Query } from "react-apollo";
+import { ApolloClient } from "apollo-client";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { HttpLink } from "apollo-link-http";
+import { BoostEmbedIframe } from "../boosts/BoostEmbedIframe";
+
+const GET_LINK_QUERY = gql`
+  query($reference: String!) {
+    postsGetByReference(reference: $reference) {
+      id
+    }
+  }
+`;
+const CREATE_LINK_MUTATION = gql`
+  mutation($input: PostCreateExternalLinkInput!) {
+    postsCreateExternalLinkEmbedded(input: $input) {
+      id
+    }
+  }
+`;
+
+// const dappOrigin = "http://localhost:3000"; // if simultaneously working on/testing dapp embed locally
+const dappOrigin = process.env.NODE_ENV === "development" ? "https://staging.civil.app" : "https://registry.civil.co";
+const fallbackFallbackUrl = dappOrigin + "/storyfeed"; // if embed doesn't load, suggest they go here
+
+const apolloClient = new ApolloClient({
```

Comment:
I assume this one-off apolloClient is so that we didn't need to bring in a bunch of other packages? (utils maybe?)
We should think about how to refactor this at some point

Reply:
Yeah this is to avoid bringing in utils which brings in a ton of stuff. We could definitely break out apollo utils into its own package which would do the trick. Though I did just realize I could change this so that we don't need apollo here at all - all the queries could be moved to inside the iframe if instead of loading embed by boost ID, we just load e.g. `/embed/boost/story?url=...` and then all the gql action can happen inside the dapp embed.